ajax/libs/gasparesganga-jquery-loading-overlay/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "gasparesganga-jquery-loading-overlay",
```

Comment:
I think we can use `jquery-loading-overlay` which is the same as the repo name.
In this case, please also update commit msg & PR title.
Thanks.